torch/library.py

```diff
@@ -110,6 +110,28 @@ def define(self, schema, alias_analysis="", *, tags=()):
         _defs.add(qualname)
         return result
 
+    def _impl_t_c(self, op_name, fn, dispatch_key=''):
```

Comment:
docs???

Reply:
Please change the name to something more readable. No one is going to know what `_impl_t_c` is (I still don't know what it is)